example.csharp/addons/tiltfive/scenes/T5ControllerCS.cs

```diff
@@ -0,0 +1,25 @@
+using Godot;
+using System;
+
+public partial class T5ControllerCS : Node3D
```

Comment:
This appears to be missing a few other functions:
- bool get_has_tracking_data()
- bool get_is_active()
- XRPose get_pose()

These are actually inherited from T5Node3D which suggests we may want a T5NodeCS class.

Reply:
Good point